scipy/stats/_generate_pyx.py

```diff
@@ -0,0 +1,16 @@
+import pathlib
+
+def make_biasedurn():
+    # numpy's random C API changes between 1.17 and 1.18
```

Comment:
This isn't quite true. The old C API remained, and a completely new one was added.

The new API in the 1.18.x releases wasn't 100% stable, and in 1.19.0 there was a change that seems relevant to the code in this PR: https://numpy.org/devdocs/release/1.19.0-notes.html#numpy-random-bit-generator-moved-to-numpy-random-bit-generator 

I think you may want to change to `< 19` here.

Reply:
Changed in https://github.com/mdhaber/scipy/pull/56